server_info.go

```diff
@@ -0,0 +1,85 @@
+package git
```

Comment:
Not sure this is the best package for `server_info` and its tests. Maybe somewhere inside `plumbing` could be a better fit?

Reply:
Moved it to `plumbing/serverinfo`